### ISSUE TITLE
wilcoxon p-value

### ISSUE THREAD

**Issue by remrama** [raphaelvallat/pingouin#30]
There's a line (245) in the `wilcoxon` function of `nonparametric.py`:
```
pval *= .5 if tail == 'one-sided' else pval
```

I _think_ this is intended to work as
```
pval = pval*.5 if tail == 'one-sided' else pval
```

but is actually doing
```
pval *= (.5 if tail == 'one-sided' else pval)
```

Easy fix if I'm right! :)

Also, far less important but while I'm here:
pingouin docs for wilcoxon says "A continuity correction is applied by default..." but this is not the case, right?? Scipy wilcoxon's default is `False`, and pingouin also explicitly passes that.

**Comment by raphaelvallat** [comment]
Thank you so much @remrama for noting this (major) error. As a result, all the two-sided p-values from the `wilcoxon` function were squared, and therefore smaller. This is now fixed in 6aa339ef6e89b3182c9e092ba1d45896b395fe85 (as well as the correction, thanks for noting that too!), and I will publish a new release ASAP.

Thanks again for your sharp eyes!
Raphael

**Comment by raphaelvallat** [comment]
Just published Pingouin 0.2.6 that fixes this error. Closing the issue. Thanks again!
